packages/azure-http-specs/specs/azure/resource-manager/operation-templates/mockapi.ts

```diff
@@ -29,10 +29,61 @@ const validOrder = {
     lastModifiedByType: "User",
   },
 };
+const validCheckNameAvailability = {
```

Comment:
nit, change the var name, e.g. checkNameAvailabilityResponse

Reply:
Fixed by the [commit#a2103e4](https://github.com/Azure/typespec-azure/pull/1934/commits/a2103e423b95bc7eb27c0fe6a77b3cb37ee6776c)